.github/workflows/docker.yml

```diff
@@ -0,0 +1,73 @@
+# Inspiration:
+# - https://docs.github.com/en/actions/guides/publishing-docker-images#publishing-images-to-github-packages
+# - https://github.com/docker/build-push-action/blob/master/docs/advanced/test-before-push.md
+name: Test the docker image
+on:
+  # TODO: publish 1/ "master" for each "master" merged PR
+  # then 2/ for each "release". This will let us iterate
+  # without consequences, and control the moment when we want to tag & ship.
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Comment comptes-tu adapter après ? Le job actuel s'appelle `build-and-push-image` mais j'imagine qu'on voudrait :
- `build` pour toutes les PR ayant comme target `master`
- build + push sur `master` ou au tag ?

Reply:
Je comptais en discuter en équipe justement après, mais dans l'idée j'aurais bien vu:
- si il y a une `release` (il y a un event associé) au sens GitHub, alors prendre le tag associé, et créer l'image avec comme nom le tag
- si c'est une PR ou un merge ou un push sur `master`, construire une image `master` comme actuellement (pour tester)

Il reste aussi la question de comment régénérer périodiquement les images pour avoir les patchs sécurité, selon quels critères (je ne sais pas encore comment ça se gère bien, dans l'idée on n'aurait pas besoin de régénérer tous les anciens tags, peut-être juste les 2 plus récents etc).

Ces points restent à explorer !